deploy/static/cert-manager-openshift-routes.yaml

```diff
@@ -113,8 +128,4 @@ spec:
               path: "/readyz"
             initialDelaySeconds: 3
             periodSeconds: 5
-            timeoutSeconds: 3
-          securityContext:
-            allowPrivilegeEscalation: false
-            runAsUser: 65534
-            runAsGroup: 65534
\ No newline at end of file
+            timeoutSeconds: 3
```

Comment:
```suggestion
            timeoutSeconds: 3

```

Reply:
I could leave the 
securityContext: {}
, then the timeoutSeconds aren't changed in order.